aspnet/docfx.json

```diff
@@ -73,5 +73,12 @@
     "template": [],
     "dest": "_site",
     "markdownEngineName": "markdig"
-  }
+  },
+"globalMetadata": {
+      "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
+      "feedback_system": "GitHub",
+      "feedback_github_repo": "MicrosoftDocs/docs-help-pr",      
```

Comment:
This shouldn't point to docs-help-pr. Also, the search scope probably shouldn't be "help-docs". Also, I think the reason recommendations aren't working is because this isn't within the "build" key. I'd just add recommendations to the existing globalMetadata section on line 38.